### DISCUSSION TITLE
Past date leaves

### DISCUSSION THREAD

**Discussion by anandthinksys** [horilla-opensource/horilla#385] [General]
I would like to apply for past date leave but the pop-up gives a warning saying "Request cannot be made for past date". And an admin user can add it but not a normal user.
Please help me in this issue. Is there any setting I need to check or am I missing something?
![image_2024_11_05T07_00_42_778Z](https://github.com/user-attachments/assets/fb7d4a4f-0f54-4709-8405-29ea0f06fc24)

**Comment by horilla-opensource** [comment]
Hi @anandthinksys ,
You can disable that option in the settings.

Settings > Leave > Restrictions > Restricts Past Date Leave Request Creation > false

**Comment by anandthinksys** [comment]
Thank you for getting back to me.
Unfortunately, I can't see any Restrictions option under the Leave menu.
![Leave options](https://github.com/user-attachments/assets/818eae9c-2e12-41cc-8eec-88318c9ed74d)

**Comment by anandthinksys** [comment]
Hello My bad, I have not updated the project. I will check after updating the files.
